flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/debezium/reader/BinlogSplitReaderTest.java

```diff
@@ -153,6 +153,8 @@ public void testReadSingleBinlogSplit() throws Exception {
                         expected.length,
                         splits.get(splits.size() - 1).getTableId());
         assertEqualsInAnyOrder(Arrays.asList(expected), actual);
+        mySqlConnection.close();
+        binaryLogClient.disconnect();
```

Comment:
`MySqlConnection#close` and `BinaryLogClient#disconnect` both may throw exceptions. If `close()` fails, `disconnect()` will be skipped consequently.

What about let `mySqlConnection` auto-close?

Reply:
If previous assert fails, the test jvm thread should be stoped. Only the job behave success, then the connection not closed will influnce later test unit.

I think only when we catch this error for Assert.error and test will execute  later code, we should pay more attention to it.
You can see binlogReader、snapshotSplitReader.close in tBinlogSplitReaderTest.java

Reply:
You're right, it's not very necessary to check them when test passed normally.